sample/sample.go

```diff
@@ -28,23 +28,26 @@ func (s *SamplerFactory) GetSamplerImplementationForDataset(dataset string) Samp
 		return nil
 	}
 
-	s.Logger.Debug().WithField("dataset", dataset).Logf("creating sampler implementation")
```

Comment:
could this line move down to the bottom before we return and do something like the default case for the log message?